auxx/Main.hs

```diff
@@ -31,7 +31,10 @@ import           Pos.WorkMode (EmptyMempoolExt, RealMode)
 import           AuxxOptions (AuxxAction (..), AuxxOptions (..), AuxxStartMode (..), getAuxxOptions)
 import           Mode (AuxxContext (..), AuxxMode, CmdCtx (..), realModeToAuxx)
 import           Plugin (auxxPlugin, rawExec)
-import           Repl (WithCommandAction(..), withAuxxRepl)
+import           Repl (WithCommandAction (..), withAuxxRepl)
+
+loggerName :: LoggerName
```

Comment:
I believe that defining a top level function that is used once only doesn't make any sense.

Reply:
Oh, I forgot to use it in another place where we still have `"auxx"` hardcoded. Will fix.